core/domain/caching_services_test.py

```diff
@@ -164,30 +166,31 @@ class CachingServicesUnitTests(test_utils.GenericTestBase):
         'ard_is_checkpoint": true, "interaction": {"solution": null, "answer_gr'
         'oups": [{"tagged_skill_misconception_id": null, "outcome": {"param_cha'
         'nges": [], "feedback": {"content_id": "feedback_2", "html": "<p>This i'
-        's great! \\u00ae\\u00ae</p>"}, "dest": "Introduction", "refresher_expl'
-        'oration_id": null, "missing_prerequisite_skill_id": null, "labelled_as'
-        '_correct": false}, "training_data": [], "rule_specs": [{"rule_type": "'
-        'Contains", "inputs": {"x": {"normalizedStrSet": ["\\u00ae\\u00ae"], "c'
-        'ontentId": "rule_input_4"}}}]}], "default_outcome": {"param_changes": '
-        '[], "feedback": {"content_id": "default_outcome", "html": ""}, "dest"'
-        ': "Introduction", "refresher_exploration_id": null, "missing_prerequis'
-        'ite_skill_id": null, "labelled_as_correct": false}, "customization_arg'
-        's": {"rows": {"value": 1}, "placeholder": {"value": {"unicode_str": "'
-        '\\ud83d\\ude0d\\ud83d\\ude0d\\ud83d\\ude0d\\ud83d\\ude0d", "content_id'
-        '": "ca_placeholder_0"}}}, "confirmed_unclassified_answers": [], "id": '
-        '"TextInput", "hints": [{"hint_content": {"content_id": "hint_3", "html'
-        '": "<p>This is a copyright character \\u00a9.</p>"}}]}, "linked_skill_'
-        'id": null, "recorded_voiceovers": {"voiceovers_mapping": {"feedback_2"'
-        ': {}, "rule_input_4": {}, "content": {}, "hint_3": {}, "default_outcom'
-        'e": {}, "ca_placeholder_0": {}}}, "classifier_model_id": null, "conten'
-        't": {"content_id": "content", "html": "<p>Unicode Characters \\ud83d\\'
-        'ude0d\\ud83d\\ude0d\\ud83d\\ude0d\\ud83d\\ude0d</p>"}, "written_transl'
-        'ations": {"translations_mapping": {"feedback_2": {}, "rule_input_4": {'
-        '}, "content": {}, "hint_3": {}, "default_outcome": {}, "ca_placeholder'
-        '_0": {}}}, "next_content_id_index": 5, "solicit_answer_details": false'
-        '}}, "version": 0, "correctness_feedback_enabled": false, "edits_allowe'
-        'd": true, "language_code": "en", "objective": "", "init_state_name": "'
-        'Introduction", "blurb": "", "author_notes": ""}'
+        's great! \\u00ae\\u00ae</p>"}, "dest": "Introduction", "dest_if_really'
```

Comment:
If you're making a change here, don't you also need to update the states_schema_version? Also, does this not go through the migration functions when it's loaded? Why do you need to explicitly add the field here?

Reply:
Right, shouldn't have been there, fixed.